configen/FileGenerator.swift

```diff
@@ -94,35 +94,72 @@ struct FileGenerator {
     }
     
     var line = ""
+    var finalType = type
+    var isObjcOptional = false
+    
+    if optionsParser.isObjC && type.contains("?") {
+      finalType = type.replacingOccurrences(of: "?", with: "")
+      isObjcOptional = value as? String == "nil" ? true : false
+    }
     
-    switch (type) {
+    switch (finalType) {
     case ("Double"):
-      line = template.doubleImplementation
+      if isObjcOptional && template is ObjectiveCTemplate{
```

Comment:
Formatting here. Space between `Template` and `{` and between `}` and `else`. Same occurs in the cases below as well.